src/services/share/deploy.ts

```diff
@@ -34,6 +35,11 @@ export const getTasksForServerSide = (appVersionId: number, directoryPath?: stri
         task: prepareEnvironmentTask,
         enabled: ctx => Boolean(ctx.showPrepareEnvironmentTask),
       },
+      {
+        title: 'Deny conflicting deployment',
+        task: handleDenyConflictingDeploymentTask,
+        enabled: ctx => Boolean(ctx.stopDeploymentTaskOnConflictError),
```

Comment:
This task should show as an error (with a ❌) in the flow and not as something that has succeeded with a ✅

Reply:
I don't have a strong opinion here

Reply:
@gregra81 I do tend to agree with Dor, showing ✅ here does not reflect what actually happened.
I think showing an error ❌  with a verbose text like I added above is a good middle-way

Reply:
Updated to error, looks better IMO